docs/en/ingest-management/fleet/fleet-settings.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[fleet-settings]]
+= {fleet} settings
+
+beta[]
+
+Configure {fleet} settings in {kib} to apply global settings to all {agent}s
+enrolled in {fleet}:
+
+. In {kib}, open the main menu, then click *Management > {fleet}*.
+
+. Click *{fleet} settings*
++
+--
+[cols="2*<a"]
+|===
+|
+[id="fleet-server-hosts-setting"]
+*{fleet-server} hosts*
+
+| The URLs your {agent}s will use to connect to a {fleet-server}. This setting
+is required. On self-managed clusters, you must specify one or more URLs. On
+{ecloud}, this field is populated automatically.
+
+//QUESTION: What is valid here? URL implies something like http://localhost:8220.
+//Is the scheme and port required or does the setting use defaults if their
+//not specified?
+
+//QUESTION: What do we need to say about IPv6 addresses?
+
+By default, {fleet-server} is exposed on the following ports:
+
+`8220`::
+Default {fleet-server} port for self-managed clusters
+
+`443`::
+Default {fleet-server} port for {ecloud}
+
+//QUESTION: Is this true? Is 443 always the default on cloud?
```

Comment:
I think we should avoid discussion of cloud 'default' for port, it is set automatically and should be left alone.  It brings another Docs need (and testing need, for 7.14) to light... that if a user deletes the Fleet Server URL in their hosted elastic cloud setup, it may be very hard to get it back right, and certainly won't be straight forward.  We should add a button to re-set it.  @mostlyjason @ruflin does that sound like a feature we'd want, and is it feasible?

Reply:
the port is not always 443 for cloud it could be `9243` too. Yes I guess a feature reset to default will be feasible and could make sense if user mess the cloud one. 
For now it's still possible to get the fleet url from the cloud console, maybe we can document that, no?

Reply:
I know I deleted it once accidentally, so it's feasible that users might need to reset it. :-) 

@nchaulet Re: your comment "...for now it's still possible to get the fleet url from the cloud console." Is that likely to change? If so, we probably should not document it.

I think it doesn't hurt to provide the ports. I can say "443 or 9243"

Reply:
> for now it's still possible to get the fleet url from the cloud console

This should not change